app/src/main/java/org/mozilla/focus/browser/cursor/CursorView.kt

```diff
@@ -36,6 +39,16 @@ class CursorView(context: Context, attrs: AttributeSet) : ImageView(context, att
         resetCountdown()
     }
 
+    fun updateCursorPressedState(event: KeyEvent) {
+        val remoteKey = RemoteKey.fromKeyEvent(event)
+        if (event.action == KeyEvent.ACTION_DOWN && (remoteKey == RemoteKey.CENTER
```

Comment:
nit: boolean operators should always be at the end of the line and you should generally group related statements. In this case:
```kotlin
        // Enter is for keyboard and emulator use.
        if (event.action == KeyEvent.ACTION_DOWN &&
                (remoteKey == RemoteKey.CENTER || event.keyCode == KeyEvent.KEYCODE_ENTER)) {
```
Assuming line length isn't too long.